net-libs/shairplay/shairplay-0_pre20170118.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+
+inherit autotools vcs-snapshot
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="git://github.com/juhovh/shairplay.git"
+else
+	EGIT_COMMIT="498bc5bcdd305e04721f94a04b9f26a7da72673f"
+	SRC_URI="https://github.com/juhovh/shairplay/archive/${EGIT_COMMIT}.tar.gz -> ${P}.tar.gz"
+fi
+
+DESCRIPTION="Apple airplay and raop protocol server"
+HOMEPAGE="https://github.com/juhovh/shairplay"
+LICENSE="BSD LGPL-2.1 MIT"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
As QA just helpfully pointed out, to use this, KEYWORDS needs to be moved into the else branch of the if 9999 conditional.